docs/concepts-broker-load-balancing-concepts.md

```diff
@@ -173,7 +173,14 @@ Based on the broker resource usage (for example, the number of topics or session
 
 Step 3: split bundles at the specific boundaries from step 2.
 
-For how to split bundles manually, see TBD (the docs is WIP, stay tuned!).
+For how to split bundles manually, please refer to [split-bundle](https://pulsar.apache.org/reference/#/3.2.x/pulsar-admin/namespaces?id=split-bundle) admin command.
+
```

Comment:
This link would point to a hardcoded versioned URL, where we should use the URL relative to each document version.

Reply:
thanks. updated